tests/functional/monitoring/prometheus/alerts/test_capacity.py

```
@@ -31,54 +31,83 @@ def test_rbd_capacity_workload_alerts(
     measure_start_time = workload_storageutilization_97p_rbd.get("start")
     measure_end_time = workload_storageutilization_97p_rbd.get("stop")
+    logger.info(
+        f"Measurement period: start={measure_start_time}, end={measure_end_time}"
+    )
 
     # Check utilization on 97%
     alerts = workload_storageutilization_97p_rbd.get("prometheus_alerts")
-    if config.ENV_DATA.get("ocs_version") >= "4.21":
+    logger.info(f"Number of alerts retrieved: {len(alerts) if alerts else 0}")
+    ocs_version = config.ENV_DATA.get("ocs_version")
+    logger.info(f"OCS version: {ocs_version}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)